utils/cliutils/utils.go

```diff
@@ -88,9 +88,17 @@ func PrintSummaryReport(success, failed int, reader *content.ContentReader, rtUr
 	log.Output(basicSummary)
 	defer log.Output("}")
 	for file := new(serviceutils.FileInfo); reader.NextRecord(file) == nil; file = new(serviceutils.FileInfo) {
+		var source, target string
+		if rtUrl != "" {
+			source = rtUrl + file.ArtifactoryPath
+			target = file.LocalPath
+		} else {
+			source = file.LocalPath
+			target = file.ArtifactoryPath
+		}
```

Comment:
I think that taking this out into a new function such as the following would a better choice here:
```
func getSourceAndTarget(file serviceutils.FileInfo, rtUrl string) (source, target string)  {
	if rtUrl != "" {
		// Upload
		source = rtUrl + file.ArtifactoryPath
		target = file.LocalPath
	} else {
		// Download
		source = file.LocalPath
		target = file.ArtifactoryPath
	}
	return
}
```